RubeesEat.IntegrationTests/WebTesting/ControlObject.cs

```diff
@@ -27,6 +29,47 @@ public IReadOnlyCollection<IWebElement> FindElements(By by)
             return Me.FindElements(by);
         }
         
+        public IWebElement FindElement(string t_id)
+        {
+            ReadOnlyCollection<IWebElement> elements = Me.FindElements(By.CssSelector($"[data-test-id='{t_id}']"));
+            if (elements.Count == 0)
+            {
+                throw new NoSuchElementException($"Element with id: {t_id} was not found.");
+            }
+
+            if (elements.Count > 1)
+            {
+                throw new TooManyElementsException();
+            }
+            return elements.First();
+        }
+        
+        public void InvokeAction(string action)
+        {
+            ReadOnlyCollection<IWebElement> elements = Driver.FindElements(By.CssSelector($"[data-test-action='{action}']"));
+            if (elements.Count == 0)
+            {
+                throw new NoSuchElementException($"Element with id: {action} was not found.");
+            }
+            if (elements.Count > 1)
+            {
+                throw new TooManyElementsException();
+            }
+            var element = elements.First();
+            var clickBehavior = element.GetAttribute("data-test-click-behavior");
+            switch (clickBehavior)
+            {
+                case "ClickAndWaitUntilStale":
+                    element.ClickAndWaitUntilStale();
+                    break;
+                case "Click":
+                    element.Click();
+                    break;
+                default:
+                    throw new UnsupportedClickBehavior(clickBehavior, action);
+            }

```

Comment:
By using switch expressions, the code becomes more concise and readable
```suggestion
            clickBehavior switch
            {
                "ClickAndWaitUntilStale" => element.ClickAndWaitUntilStale(),
                "Click" => element.Click(),
                _ => throw new UnsupportedClickBehavior(clickBehavior, action);
            };
```

Reply:
Unfortunately, the code has an error if i use that. the ; on line 64 is wrong, and if i remove it, it is still an error because switch expressions only work with expressions, not statements.